src/transform/src/index.ts

```diff
@@ -3,14 +3,43 @@ import { createWriteStream } from "fs";
 import { HypermodeMetadata } from "./metadata.js";
 import { Extractor } from "./extractor.js";
 import binaryen from "assemblyscript/lib/binaryen.js";
+import {
+  Parser,
+  FunctionDeclaration,
+  IdentifierExpression,
+  NodeKind,
+  CommonFlags,
+} from "assemblyscript/dist/assemblyscript.js";
+import { isStdlib } from "./utils.js";
 
 export default class HypermodeTransform extends Transform {
+  public embedders: string[] | null = null;
+  afterParse(parser: Parser): void | Promise<void> {
+    for (const source of parser.sources) {
+      if (source.isLibrary) continue;
```

Comment:
Would this prevent us from making a library of embedders, or including some default embedders in the existing `@hypermode/functions-as` library?

Reply:
You can only export from the entry file. It *should* handle cases like `export { embedderFunction } from "../path-to-embedder.ts";`
I should check that

Reply:
Okay, it doesn't work. Adding support now

Reply:
Done.